proto/src/warden.proto

```diff
@@ -19,12 +19,15 @@ service Warden {
 }
 
 message FullAssignment {
-   repeated RangeId range = 1;
+   int64 version = 1;
+   repeated RangeId range = 2;
 }
 
 message IncrementalAssignment {
-    repeated RangeId load = 1;
-    repeated RangeId unload = 2;
+    int64 version = 1;
+    int64 previous_version = 2;
```

Comment:
Could you please add some documentation as to what these versions mean? (either here or somewhere in the warden package). I have some guesses as to how they're supposed to work but I'm not sure :)

Reply:
Done.